CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Bob versions changelog
 #### Added
 - Add clusterwide delete operation (#364)
 - TLS support, TLS for grpc or rest can be enabled via cluster & node config (#303)
+- Final summary and return code to `bobt`
```

Comment:
```suggestion
- Final summary and return code to `bobt` (#649)
```

Reply:
Fixed in [7776e09](https://github.com/qoollo/bob/pull/652/commits/7776e0984961e83d005f56f2f450f68571fed689)